main.rb

```diff
@@ -0,0 +1,9 @@
+def foo
+  eval(foo)
+end
+
+class AAA
+        def initialize
```

Comment:
__[rubocop]__
* Missing top-level class documentation comment.
* Use 2 (not 8) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)

<!-- Comment by [SideCI](https://sideci.com) -->